src/app/learnocaml_index_main.ml

```diff
@@ -706,6 +727,10 @@ let set_string_translations () =
     "txt_returning", [%i"Returning user"];
     "txt_returning_dialog", [%i"Enter your token"];
     "txt_login_returning",  [%i"Connect"];
+    "txt-logout", [%i "Logout"];
```

Comment:
Why do you use "-" instead of "_" in the identifiers here?

Reply:
Well, I suppose we should use "_" for identifiers of internationalized strings and and "-" for the rest? As far as I remember, this was not really consistent across the entire codebase actually (sometimes classes use one or the other, and ids too). Anyway, I'll change these ones ;)

Reply:
Thanks! Let's not propagate chaos :)